Tests/Parser/Client/fixtures/browser.yml

```diff
@@ -2647,3 +2647,11 @@
     version: "6.0"
     engine: WebKit
     engine_version: "537.36"
+-
+  user_agent: Mozilla/5.0 (Linux; Android 5.1; T8 Build/LMY47D) AppleWebKit/537.36 (KHTML, like Gecko) YoloBrowser/1.0.1.74 Safari/537.36
+  client:
+    type: browser
+    name: YOLO Browser
+    version: 1.0.1.74
+    engine: WebKit
```

Comment:
change `WebKit`to `Blink`

The Webkit engine was used until the 28th version of chrome, after which it is considered Blink; the Only place where WebKit remained is MacOs/xOS.